.github/workflows/build.yaml

```diff
@@ -119,15 +119,34 @@ jobs:
       - run: ls -R artifacts
       - run: zip -r ../../${{ env.project }}.windows.x86_64.zip *
         working-directory: artifacts/${{ env.project }}.windows.x86_64
+      - name: tag prerelease
```

Comment:
What is this needed for? Looking at the action source, it seems to force update a git tag

Reply:
Releases in github have to point to a tag.  The strategy I've been using is to have a release named "latest", and an associated tag which moves with the head of the master branch.  It's useful for maintaining a snapshot release without cluttering up the git tags to have a new one on each commit or merge.